sonar-db/src/main/java/org/sonar/db/DatabaseChecker.java

```diff
@@ -43,12 +44,29 @@ public DatabaseChecker(Database db) {
   @Override
   public void start() {
     try {
-      if (H2.ID.equals(db.getDialect().getId())) {
-        Loggers.get(DatabaseChecker.class).warn("H2 database should be used for evaluation purpose only");
-      } else if (Oracle.ID.equals(db.getDialect().getId())) {
-        checkOracleVersion();
+      switch (db.getDialect().getId()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)